src/styles/index.scss

```diff
@@ -1,3 +1,69 @@
+$gap: 1px;
+$day-height: 100px;
+$day-width: 100px;
+$days-week: (1 'mon', 2 'tue', 3 'wed', 4 'thu', 5 'fri', 6 'sat', 7 'sun');
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+}
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  gap: $gap;
+  width: calc(7 * ($day-width + $gap));
+  flex-wrap: wrap;
+
+  &__day {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    height: $day-height;
+    width: $day-width;
+    background-color: #eee;
+    border: 1px solid #000;
+    font-size: 30px;
+
+    transition-duration: 0.5s;
```

Comment:
It is better not to set transitions for everything, it affects performance. it's better to be specific.